src/core/textures/VideoBaseTexture.js

```diff
@@ -54,13 +55,8 @@ export default class VideoBaseTexture extends BaseTexture
         this.width = source.videoWidth;
         this.height = source.videoHeight;
 
-        /**
-         * Should the base texture automatically update itself, set to true by default
-         *
-         * @member {boolean}
-         * @default true
-         */
-        this.autoUpdate = false;
+        this._autoUpdate = true;
+        this._isAutoUpdating = false;
```

Comment:
I had 2 because i was worried about a certain set of circumstances that could lead to the ticker being joined twice, such as using the autoPlay setter before the canplay comes through from the video source. Thus one flag for the getter/setter on whether it _should_ autoplay, and another for if it _is_ autoplaying